doc/spire_agent.md

```diff
@@ -45,7 +45,7 @@ This may be useful for templating configuration files, for example across differ
 | `log_format`              | Format of logs, \<text\|json\>                                        | Text                 |
 | `server_address`          | DNS name or IP address of the SPIRE server                            |                      |
 | `server_port`             | Port number of the SPIRE server                                       |                      |
-| `socket_path`             | Location to bind the workload API socket                              | $PWD/spire_api       |
+| `socket_path`             | Location to bind the workload API socket                              | /tmp/agent.sock      |
```

Comment:
The default value described in agent_full.conf needs to be updated also.
https://github.com/spiffe/spire/blob/master/conf/agent/agent_full.conf#L31

Reply:
Got it, thanks!